packages/medusa-fulfillment-webshipper/src/services/__tests__/webshipper-fulfillment.js

```diff
@@ -323,5 +323,101 @@ describe("WebshipperFulfillmentService", () => {
         tarif_number: "test",
       })
     })
+
+    it("builds a webshipper item with weight from product variant", async () => {
+      const webshipper = new WebshipperFulfillmentService(
+        {
+          totalsService,
+        },
+        {}
+      )
+
+      medusaItem.variant = {
+        weight: 1234
+      }
+
+      medusaItem.variant.product = {}
+
+      let item
+      try {
```

Comment:
I think we can omit the `try/catch` in the test if it's not expected

Reply:
good point, just copy pasted the boilerplate from a similar test case.